scenarios/tracking/02_evaluating_mot_challenge.ipynb

```diff
@@ -0,0 +1,459 @@
+{
```

Comment:
<ol><li>Do you still want TRACK_BUFFER to be 300? It makes the model run pretty slowly. Apparently this was necessary for carcans but if it's not needed for MOTChallenge data, maybe we can consider bumping it down.</li><li>reseult_root =&gt; result_root</li><li>all_dla34.pth is the baseline model, so consider using a var name like BASELINE_MODEL instead of FT_MODEL</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Will use TRACK_BUFFER = 30 the default value. Actually I didn't see the model run really slowly.

Changed other points.